docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -518,6 +520,162 @@ In order to see the current maintenance state on a node:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start the server, logon to the ssh terminal and make a first checkup of the services:
+PacketFence-Mariadb, type the command: ::
+----
+ systemctl status packetfence-mariadb
+----
+----
+ Loaded: loaded (/usr/lib/systemd/system/packetfence-mariadb.service; enabled; vendor preset: disabled)
+ Active: active (running) since ...
+----
+
+PacketFence-Config, type the command: ::
+----
+ systemctl status packetfence-config
+----
+----
+ Loaded: loaded (/usr/lib/systemd/system/packetfence-config.service; enabled; vendor preset: disabled)
+ Active: active (running) since ...
+----
+
+HA-Proxy, type the command: ::
+----
+ systemctl status packetfence-haproxy
+----
+----
+ Loaded: loaded (/usr/lib/systemd/system/packetfence-haproxy.service; enabled; vendor preset: disabled)
+  Active: active (running) since ...
+----
+
+The *Master* must now be ready.
+
+
+Bring up the "Slaves"
+^^^^^^^^^^^^^^^^^^^^^
+
+Start the first slave, logon to the ssh terminal.
+
+Check the PacketFence-Mariadb sync with the Master, type the command: ::
+----
+mysql -u root -p
+mysql> show status like 'wsrep%';
+----
+----
+MariaDB [(none)]> show status like "wsrep%";
++------------------------------+-------------------------------------------------------+
+| Variable_name                | Value                                                 |
++------------------------------+-------------------------------------------------------+
+...
+| wsrep_cluster_size           | 2                                                     |
+...
+| wsrep_connected              | ON                                                    |
+...
+| wsrep_evs_state              | OPERATIONAL                                           |
+...
+| wsrep_local_state_comment    | Synced                                                |
+...
++------------------------------+-------------------------------------------------------+
+----
+
+Start the second slave, logon on terminal and go with a *show status like 'wsrep%';* once again.
+
+* The values must have changed to:
+----
+MariaDB [(none)]> show status like "wsrep%";
++------------------------------+-------------------------------------------------------+
+| Variable_name                | Value                                                 |
++------------------------------+-------------------------------------------------------+
+...
+| wsrep_cluster_size           | 3                                                     |
+...
+| wsrep_evs_state              | OPERATIONAL                                           |
+...
+| wsrep_local_state_comment    | Synced                                                |
+...
++------------------------------+-------------------------------------------------------+
+----
+
+NOTE: The _wsrep_incoming_addresses_ will give you the IP addresses of the nodes synced.
+
+NOTE: The _wsrep_cluster_status_ will always be *Primary*, even on the slaves. 
+
+'''
+Backup procedure
```

Comment:
Backups don't only apply to clusters, so this should be moved in the admin guide

Reply:
Are we moving this in the admin guide?